Dockerfile

```
@@ -0,0 +1,15 @@
+FROM python:3.7-alpine
+MAINTAINER Josep Lagunas Calpe
+
+ENV PYTHONUNBUFFERED 1
+
+COPY requirements.txt /requirements.txt
+
+RUN pip install -r /requirements.txt
+
+RUN mkdir /app
+WORKDIR /app
+COPY ./app /app
```

I recommend using this approach instead of copying all the code.

```suggestion
ENV PYTHONPATH /app/
ADD . /app/
```

---

I am getting an error running "python manager.py ...". I will check the documentation and for sure consider using it instead of COPY from now on.